homeassistant/components/panel_iframe/__init__.py

```diff
@@ -2,7 +2,7 @@
 import voluptuous as vol
 
 from homeassistant.components import frontend
-from homeassistant.const import CONF_ICON, CONF_URL
+from homeassistant.const import CONF_ICON, CONF_URL, CONF_ALLOW
```

Comment:
I don't think we should add `CONF_ALLOW` at global `homeassistant.const` as it's only used here.

Add `CONF_ALLOW` locally, as `CONF_TITLE` do.